### DISCUSSION TITLE
You know OBS has a $3000 bounty for someome to implement VST3...it would be great if whoever designed it made it work for LV2 in addition to VST3

### DISCUSSION THREAD

**Discussion by ericfont** [DISTRHO/DPF#347] [General]
$3000 OBS VST3 bounty here: https://github.com/obsproject/obs-studio/discussions/5074

I bring this up here cause maybe someone involved with DPF has good knowledge on how to implement the OBS plugin in a way that works for both LV2 and VST3.

**Comment by falkTX** [comment]
Good to know, but this is unrelated to DPF since it is about plugin hosting inside OBS, not OBS as a plugin.

**Comment by ericfont** [comment]
yes I understand it is unrelated, which is why I put this message in a "General chat"...you can ignore it, sorry.  Just wanted to put it out as an off-topic chat, cause maybe one of the contributors to DPF might have the right skill set and be interested.

**Reply by falkTX** [reply]
It is for sure interesting, I mention just in case you thought it was 100% related to DPF.

I need to see how these bounties typically go. I am doing a custom non-vst3sdk implementation for DPF, which can be suitable for OBS.
Though before that happens I would very likely do it in Carla first as a way of testing and getting to know the host-side too.

Thanks in any case, don't be afraid to post such links in the future, they are useful
